avm/res/storage/storage-account/main.bicep

```diff
@@ -561,7 +701,7 @@ type networkAclsType = {
 
   @description('Required. Specifies the default action of allow or deny when no other rules match.')
   defaultAction: ('Allow' | 'Deny')
-}
+}?
```

Comment:
noooooooooooooooooooooooooooo

Reply:
😱

Reply:
@fblix just fixed it : https://github.com/Azure/bicep-registry-modules/pull/1698

That change was not supposed to be in here and was a sneaky one at that.